lib/matplotlib/mpl-data/stylelib/classic.mplstyle

```diff
@@ -36,7 +36,7 @@ patch.force_edgecolor  : True
 patch.edgecolor        : k
 patch.antialiased      : True    # render patches in antialiased (no jaggies)
 
-hatch.color            : k
+hatch.color            : inherit
```

Comment:
This should not change, the point of this is to look exactly like mpl 1.x.

Reply:
Here, I tried to follow the logic similar to the one mentioned by @story645 in this [comment](https://github.com/matplotlib/matplotlib/pull/26993#issuecomment-1967465022)

If it was set to k, then we wouldn't be able to use the if statements that check for inherit